packages/core/mutations/cancelOfferTransaction.graphql

```diff
@@ -1,7 +1,7 @@
 # This query should use the mutation `cancelOfferTransaction` to cancel an offer
-query OfferWithCancel($offerId: UUID!, $account: Address!) {
-  offer(id: $offerId) {
-    cancel(account: $account) {
+mutation CreateCancelOfferTransaction($offerId: String!, $account: Address!) {
```

Comment:
Why is offerId a string when it is everywhere a uuid

Reply:
Indeed it should have been a UUID. The mutation `createCancelOfferTransaction` only accept a string. It will requires a breaking change to update this. I will put a todo to change it in API v1.